committee-steering/governance/sig-governance.md

```diff
@@ -273,7 +273,7 @@ Issues impacting multiple subprojects in the SIG should be resolved by either:
 [The Release Team: Bug Triage, CI Signal, and more]: https://github.com/kubernetes/sig-release/tree/master/release-team/role-handbooks
 [Production Readiness Reviewer]: /sig-architecture/production-readiness.md#becoming-a-prod-readiness-reviewer-or-approver
 [API Reviewer and Moderator]: /sig-architecture/api-review-process.md#expanding-the-reviewer-and-approver-pool
-[Marketing Council]: /communication/marketing-team/role-handbooks/council.md
+[Marketing Council]: /communication/contributor-comms/role-handbooks/council.md
```

Comment:
I think you'll have to make this change in sigs.yaml: https://github.com/kubernetes/community/blob/master/sigs.yaml and then run `make` in the root of the repo to generate this change in this file. That's why the verification job is failing I think

Reply:
I think I've commented on the wrong line, not sure what happened, but please give me a minute and I'll correct the error!

Reply:
Commented: https://github.com/kubernetes/community/pull/6714#discussion_r956186363